core/unit_test/tools/printing-tool.cpp

```diff
@@ -0,0 +1,110 @@
+
+#include <cstdio>
+#include <inttypes.h>
+#include <vector>
+#include <string>
```

Comment:
You are not using `std::string` here, do you?
```suggestion
```

Reply:
Addressed, thanks